### PR TITLE
Dnyaneshwar html file

### PR DESCRIPTION
**Pull request by dnyaneshwarkhule**
complete my task added my file

### CODE REVIEW

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
please make sure all files

File: dnyaneshwar k/mi mean.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    <head>
+
+        <titile> MI-MEAN9 </titile>
+    </head>
+
+    <body>
+        there are 4 member in my batch including me
+    
+
+    </body>
```
Comment (Pawan1005) [inline dnyaneshwar k/mi mean.html:14]: check file is correct or not before pushing on git hub

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
make indentation properly in every file 
remove extra line

**Comment by dnyaneshwarkhule** [comment]
Ok sir

**Review by MKadeer17 — APPROVED** [review]

File: dnyaneshwar k/dnyaneshwarA3_1.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    <head>
+
+        <titile> list tag </titile>
+    </head>
+
+    <body>
+        <h3 style="text-align:left"> Ambersoft</h3>
+
+    </body>
+
```
Comment (MKadeer17) [inline dnyaneshwar k/dnyaneshwarA3_1.html:14]: Order list comes in body tag not outside the body tag.

File: dnyaneshwar k/mi mean.html (hunk 1 above)
Comment (MKadeer17) [inline dnyaneshwar k/mi mean.html:14]: closing html tag missing.

**Pull request closed without merging**